services/processor/native/repository/Committee/committee.go

```diff
@@ -7,14 +7,14 @@
 package committee_systemcontract
 
 import (
-	"bytes"
 	"encoding/binary"
 	"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/env"
 	"github.com/orbs-network/orbs-network-go/crypto/hash"
-	"math"
-	"sort"
 )
 
+/**
+ * This function is meant ot be used via the callsystemcontract func ... it will not give same result when used with RunQuery
```

Comment:
Why won't it give the same result?

Reply:
When you use callSystemContract its a very special case of runQuery where the state is done for the last committed block but the env is for current suggested block. Since we use the current block number as our random seed this changes the order of the committee ... this is all done on purpose.

Reply:
@noambergIL maybe we can add some way to check whether we're running from `callSystemContract` and panic otherwise?

Reply:
@noambergIL cool, thanks!

Reply:
@electricmonk i don't think we want to add an env that tells you if you are runquery/sendtx/callsystem.